### DISCUSSION TITLE
cannot import name 'IdResolver' from 'atproto'

### DISCUSSION THREAD

**Discussion by venteto** [MarshalX/atproto#237] [Questions] [answered]
I'm trying to follow the [example here](https://atproto.blue/en/latest/atproto_identity/index.html) to implement a save method in a Django model that will fetch a user's DID when given their handle. But even though atproto is installed in my venv (and working with Flask), I get this error when I try to run the Django devel server:

```
.../models.py", line 1, in <module>
    from atproto import IdResolver
ImportError: cannot import name 'IdResolver' from 'atproto' 
```

Not sure if I'm just overlooking somethig obvious.

**Comment by MarshalX** [comment, marked as answer]
The minimum required version for the identity module is 0.0.36. what is your atproto SDK version? pls show the output of `pip show atproto`

to update SDK use `pip install -U atproto`

**Reply by venteto** [reply]
Aha, I'll try updating ... 

edit: yay, it works now! Thank you!

This was the output:
```
Name: atproto
Version: 0.0.35
Summary: The AT Protocol SDK
Home-page: https://github.com/MarshalX/atproto
Author: Ilya (Marshal)
Author-email: ilya@marshal.dev
License: MIT
Location: /home/ddg/.py_virtual_envs/feed_gen_at_proto_py_sdk/lib/python3.11/site-packages
Requires: click, httpx, libipld, pydantic, pyjwt, typing-extensions, websockets
Required-by: 
```
